upgrade_test.py

```diff
@@ -470,200 +473,216 @@ def test_rolling_upgrade(self):  # pylint: disable=too-many-locals,too-many-stat
+        # the data lose.
+        # But the execute time of workloads are not exact, so let only use basic prepare write & read verify for
+        # complex workloads,and comment two complex workloads.
+        #
+        # TODO: retest commented workloads and decide to enable or delete them.
+        #
+        # complex workload: verify data by multiple ops
+        # self.log.info('Starting c-s complex workload to verify data by multiple ops')
+        # stress_cmd_complex_verify_more = self.params.get('stress_cmd_complex_verify_more')
+        # complex_cs_thread_pool = self.run_stress_thread(stress_cmd=stress_cmd_complex_verify_more,
+        #                                                profile='data_dir/complex_schema.yaml')
+
+        # wait for the complex workload to finish
+        # self.verify_stress_thread(complex_cs_thread_pool)
+
+        # complex workload: verify data by delete 1/10 data
+        # self.log.info('Starting c-s complex workload to verify data by delete')
+        # stress_cmd_complex_verify_delete = self.params.get('stress_cmd_complex_verify_delete')
+        # complex_cs_thread_pool = self.run_stress_thread(stress_cmd=stress_cmd_complex_verify_delete,
+        #                                                profile='data_dir/complex_schema.yaml')
+        # wait for the complex workload to finish
+        # self.verify_stress_thread(complex_cs_thread_pool)
 
-            self.log.info('all nodes were upgraded, and last workaround is verified.')
+        # During the test we filter and ignore some specific errors, but we want to allow only certain amount of them
+        step = 'Step9 - Search for errors that we filter during the test '
+        self.log.info(step)
+        self.log.info('Checking how many failed_to_load_schem errors happened during the test')
+        error_factor = 3
+        schema_load_error_num = self.count_log_errors(search_pattern='Failed to load schema version',
+                                                      step=step)
+        # Warning example:
+        # workload prioritization - update_service_levels_from_distributed_data: an error occurred while retrieving
+        # configuration (exceptions::read_failure_exception (Operation failed for system_distributed.service_levels
+        # - received 0 responses and 1 failures from 1 CL=ONE.))
+        workload_prioritization_error_num = self.count_log_errors(search_pattern='workload prioritization.*read_failure_exception',
+                                                                  step=step, search_for_idx_token_error=False)
+
+        self.log.info(f'schema_load_error_num: {schema_load_error_num}; '
+                      f'workload_prioritization_error_num: {workload_prioritization_error_num}')
+
+        # Issue #https://github.com/scylladb/scylla-enterprise/issues/1391
+        # By Eliran's comment: For 'Failed to load schema version' error which is expected and non offensive is
```

Comment:
The schema load error is harmless, but it's a problem if the error is too much during the upgrade stage.

So we count the error during the whole test, and assert it in the end.